src/ansys/units/dimensions.py

```diff
@@ -143,33 +157,16 @@ def __bool__(self):
 
     def __gt__(self, __value):
         if self != __value:
-            raise DimensionsError.INCOMPARABLE_DIMENSIONS(self, __value)
+            raise IncomparableDimensions(self, __value)
 
     def __ge__(self, __value):
         if self != __value:
-            raise DimensionsError.INCOMPARABLE_DIMENSIONS(self, __value)
+            raise IncomparableDimensions(self, __value)
 
     def __lt__(self, __value):
         if self != __value:
-            raise DimensionsError.INCOMPARABLE_DIMENSIONS(self, __value)
+            raise IncomparableDimensions(self, __value)
 
     def __le__(self, __value):
         if self != __value:
-            raise DimensionsError.INCOMPARABLE_DIMENSIONS(self, __value)
-
-
-class DimensionsError(ValueError):
-    """Custom dimensions errors."""
-
-    def __init__(self, err):
-        super().__init__(err)
-
-    @classmethod
-    def INCORRECT_DIMENSIONS(cls):
-        """Return in case of dimensions not in dimension order."""
-        return cls(f"The `dimensions` key must be a 'BaseDimensions' object")
-
-    @classmethod
-    def INCOMPARABLE_DIMENSIONS(cls, dim1, dim2):
-        """Return in case of dimensions not being equal."""
-        return cls(f"The dimensions `{dim1}` cannot be compared to `{dim2}`")
+            raise IncomparableDimensions(self, __value)
```

Comment:
I'm going to create an issue to handle these in a different way rather than exposing these operators here.

Reply:
... @hpohekar I have created an issue here https://github.com/ansys/pyansys-units/issues/168

Reply:
Done. Thank you.